inc/admin/namespace.php

```diff
@@ -191,9 +191,9 @@ function admin_page() {
 				<tbody>
 					<?php foreach ( $certificate['DomainValidationOptions'] as $domain ) : ?>
 						<tr>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Name'] ) ?></td>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Type'] ) ?></td>
-							<td><?php echo esc_html( $domain['ResourceRecord']['Value'] ) ?></td>
+							<td><?php echo esc_html( isset( $domain['ResourceRecord']['Name'] ) ? $domain['ResourceRecord']['Name'] : '' ) ?></td>
```

Comment:
`$domain['ResourceRecord']['Name'] ?? ''` would be more succinct and clearer here I think.

Reply:
Didn't used it just for old version php compatibility. @rmccue Do let me know if I should still update it.

Kindest Regards,

Reply:
Updated :)